iothub/device/src/Authentication/ModuleAuthenticationWithSakRefresh.cs

```diff
@@ -10,33 +10,36 @@ namespace Microsoft.Azure.Devices.Client
     // Implementing SAS Token refresh based on a SharedAccessKey (SAK).
     internal class ModuleAuthenticationWithSakRefresh : ModuleAuthenticationWithTokenRefresh
     {
-        private readonly ClientConfiguration _clientConfiguration;
+        private readonly string _sharedAccessKey;
+        private readonly string _sharedAccessKeyName;
 
         internal ModuleAuthenticationWithSakRefresh(
             string deviceId,
             string moduleId,
-            ClientConfiguration clientConfiguration,
+            string sharedAccessKey,
+            string sharedAccessKeyName = default,
             TimeSpan sasTokenTimeToLive = default,
             int sasTokenRenewalBuffer = default)
             : base(
-                deviceId,
-                moduleId,
-                sasTokenTimeToLive,
-                sasTokenRenewalBuffer)
+                  deviceId,
```

Comment:
This was already indented 4 spaces. Why add more?

Reply:
Probably another copy-paste side-effect. I'll take another look at the spacing here.